data/data-pipeline/data_pipeline/content/config/generate_data_dictionary.py

```diff
@@ -0,0 +1,185 @@
+"""
+Quick script to generate a data dictionary for the download packet.
+
+A few notes:
+(1) This will ONLY crosswalk the downloadable CSV and the downloadable EXCEL file.
+    The SHP file has a LOT more columns, and we should almost certainly
+    make a data dictionary by hand once the column names are final.
```

Comment:
Well, I'm not sure it makes sense to give a much more robust codebook for the shapefile than the one we already provide (which has plain english and the column name). But, if we do, and we wanted to automate, it would probably make sense to wait until after the refactor.

Reply:
To address this -- 
> I'm wondering if perhaps there's a simpler way to approach this task in the code. I don't have an immediately obvious suggestion, but I am finding the code rather hard to follow for what is mostly a straightforward task. Maybe now that you have some experience with the inputs and outputs of this function, you can brainstorm a little bit the simplest, cleanest way to do the transformation? I'm mostly worried about how easy it would be for any developer / data scientist to understand and update this code.

I tried to simplify this a little bit, largely by putting it in `generate_score_post` and leveraging some of our existing utilities / code. So it's a bit dryer. I also tried to update the variable names / comments to make the operations clearer. 

The code itself is quite simple. Please let me know if these changes make it more understandable.